ci-operator/config/backube/volsync/backube-volsync-main.yaml

```diff
@@ -32,6 +98,29 @@ tests:
   commands: ./.ci-scripts/pre-commit.sh
   container:
     from: src
+- as: e2e-openshift
+  optional: true
```

Comment:
@JohnStrunk the `optional: true` means `passing e2e-openshift not required for merge` - we also have the option of setting `always_run: false` in the job. With that, the test will only run if/when somebody triggers it with a GH comment, `/test e2e-openshift` as it is now, this test will always run with PRs, but will not block merges